src/tests/DomainHost.t.sol

```diff
@@ -181,10 +181,10 @@ contract DomainHostTest is DssTest {
         host.deny(address(this));
 
         bytes[] memory funcs = new bytes[](6);
-        funcs[0] = abi.encodeWithSelector(EmptyDomainHost.lift.selector, 0);
-        funcs[1] = abi.encodeWithSelector(DomainHost.release.selector, 0);
-        funcs[2] = abi.encodeWithSelector(DomainHost.accrue.selector, 0);
-        funcs[3] = abi.encodeWithSelector(EmptyDomainHost.rectify.selector);
+        funcs[0] = abi.encodeWithSelector(EmptyDomainHost.lift.selector, 0, 0, 0);
+        funcs[1] = abi.encodeWithSelector(DomainHost.release.selector, 0, 0, 0);
+        funcs[2] = abi.encodeWithSelector(DomainHost.accrue.selector, 0, 0, 0);
+        funcs[3] = abi.encodeWithSelector(EmptyDomainHost.rectify.selector, 0, 0, 0);
```

Comment:
```suggestion
        funcs[0] = abi.encodeWithSelector(EmptyDomainHost.lift.selector, 0);
        funcs[1] = abi.encodeWithSelector(DomainHost.release.selector, 0);
        funcs[2] = abi.encodeWithSelector(DomainHost.accrue.selector, 0, 0);
        funcs[3] = abi.encodeWithSelector(EmptyDomainHost.rectify.selector, 0);
```

Why are we adding extra 0s? They do not do anything, but any reason?

Reply:
It's just for padding out all the arguments. As you said they do nothing, so it's easier to copy + paste not having to determine how many arguments a function has.

Reply:
Hmm yeah ok, it will work in the meanwhile we do not have a function with more arguments than 3.